doc/getting_started/getting_started.rst

```diff
@@ -152,6 +160,20 @@ follow the steps below to build with any custom or 3rd party cross-compilers:
 
       You only need to do this once after cloning the git repository.
 
+Now that the host tools are installed, a 3rd party cross compiler must
+be installed. See :ref:`below <third_party_x_compilers>` for
+installing a cross compiler.
+
+.. _third_party_x_compilers:
+
+Using Custom and 3rd Party Cross Compilers
+==========================================
+
+To use a 3rd party cross compiler that is not provided by the Zephyr
+SDK, follow the steps below. It is possible to use a 3rd party cross
+compiler and still use the Zephyr SDK's host tools, see :ref:`the
```

Comment:
change the comma to a period and start a new sentence:

    SDK's host tools.  See

Reply:
Done